### PR TITLE
Add microcodes for Intel and AMD processors

### PR DESCRIPTION
**Pull request by klausenbusk**
"Processor manufacturers release stability and security updates to the
processor microcode. These updates provide bug fixes that can be
critical to the stability of your system. Without them, you may
experience spurious crashes or unexpected system halts that can be
difficult to track down."[1]

[1] https://wiki.archlinux.org/index.php?title=Microcode&oldid=688966

### CODE REVIEW

**Comment by klausenbusk** [comment]
Tests are good:
```
NOTE: Executing Tasks
RESULTS:
RESULTS - config.ConfigPartitionTest.test_config_present: PASSED (0.38s)
RESULTS - kubeadm.KubeadmInitTest.test_kubeadm_init: PASSED (59.92s)
SUMMARY:
mukube-test-image () - Ran 2 tests in 60.305s
mukube-test-image - OK - All required tests passed (successes=2, skipped=0, failures=0, errors=0)
NOTE: Tasks Summary: Attempted 4099 tasks of which 4048 didn't need to be rerun and all succeeded.
```

**Review by skovbakke — COMMENTED** [review]
In general it looks good. Only a single comment.

File: meta-k8s-setup/recipes-core/microcode/amd-microcode_20210919.bb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+S = "${WORKDIR}/git"
+
+COMPATIBLE_HOST = "(i.86|x86_64).*-linux"
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+inherit deploy
+
+do_install () {
+	:
+}
```
[the remark this answers is not in the document]
Comment (klausenbusk) [reply meta-k8s-setup/recipes-core/microcode/amd-microcode_20210919.bb]: Seems to work without this noop function, removed.

**Review by skovbakke — APPROVED** [review]
LGTM

**Pull request merged**